example-backend/src/main/kotlin/app/truid/example/examplebackend/TruidSignupFlow.kt

```diff
@@ -114,7 +117,7 @@ class TruidSignupFlow(
                 val body = LinkedMultiValueMap<String, String>()
                 body.add("grant_type", "authorization_code")
                 body.add("code", code)
-                body.add("redirect_uri", "$publicDomain/truid/v1/complete-signup")
+                body.add("redirect_uri", redirectUri)
```

Comment:
A real implementation would use the same redirect URL regardless of if the user uses an app or a webpage. So it is a bit unfortuante that we need to have different URLs in the example.

Reply:
Is it possible to use a `https` URL without binding it to any domain? Would it still open the app, but it would ask the user for confirmation? If so, maybe that would be better?
We can add a note in the readme that you need to bind it to your domain, otherwise it will ask to confirm. And link to the Andorid/iOS docs.